tests/ios/Podfile.lock

```diff
@@ -569,11 +569,11 @@ PODS:
     - Firebase/MLVisionTextModel (= 6.28.1)
     - React
     - RNFBApp
-  - RNFBPerf (7.2.2):
+  - RNFBPerf (7.2.3):
     - Firebase/Performance (= 6.28.1)
     - React
     - RNFBApp
-  - RNFBRemoteConfig (7.2.2):
+  - RNFBRemoteConfig (8.0.0):
```

Comment:
Does this imply that the whole test project should be updated via Podfile / package.json as well?

Reply:
tis a good question. My impression was that running `pod install` in ios directory was the best way to keep the lockfile up to date, but I'll defer that wisdom to @Salakar

Reply:
RNFB pod versions are importing the `package.json` file and reading the version from there, no manual updates necessary. `pod install` in the tests/ios directory is all that is needed